api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ObjectController.java

```diff
@@ -249,6 +261,30 @@ public ModelAndView recordRdf(@PathVariable String collectionId,
         return (ModelAndView) handleRecordRequest(RecordType.OBJECT_RDF, data, response);
     }
 
+    /**
+     * Retrieve a record in Turtle format
+     *
+     * @param collectionId   ID of data collection or data set
+     * @param recordId       ID of record, item - a.k.a. 'localId'
+     * @param wskey          pre-api term for 'apikey'
+     * @param webRequest
+     * @param servletRequest
+     * @param response
+     * @return matching records in the turtle format
+     * @throws EuropeanaException
+     */
+    @ApiOperation(value = "get single record in turtle format)", nickname = "getSingleRecordTurtle")
+    @GetMapping(value = "/{collectionId}/{recordId}.ttl", produces = {MEDIA_TYPE_TURTLE, MEDIA_TYPE_TURTLE_TEXT, MEDIA_TYPE_TURTLE_X} )
```

Comment:
Shouldn't this be 'consumes' instead of produces?  And set 1 value for produces (MEDIA_TYPE_TURTLE)

Reply:
consumes should be used if there is a payload, as it will force us to pass content-type in the request then.  This way we can pass accept headers values: “text/turtle”,  “application/turtle” or “application/x-turtle”. If not passed it will default. For response content-type, we define that in .jsp file where we put the model and view together.